qnapstats/qnap_stats.py

```diff
@@ -57,10 +57,18 @@ def _login(self):
         data = {"user": self._username, "pwd": self._password}
         result = self._execute_post_url("authLogin.cgi", data, False)
 
+        if result is None or not result.get("authSid"):
+            # Another method to login
+            suffix_url = f"authLogin.cgi?user={self._username}&pwd={self._password}"
```

Comment:
I think CI is complaining about this line:

```
./qnapstats/qnap_stats.py:62:84: E999 SyntaxError: invalid syntax
```

I don't know enough about Python to understand why, though.

Reply:
i fix tests, i added file in wrong folder.